content/user/template/what_is_a_template.md

```diff
@@ -42,6 +42,17 @@ See [the list of non-overridable files](/user/template/template_overrides/#what-
 ### What is template default? 
 See [the template default FAQ](/user/template/template_default/). 
 
+---
+### What is responsive classic? 
+In Zen Cart 1.5.5, a new template was developed in response to the 
+increasing adoption of mobile devices like phones and tablets. Its name 
+was Responsive Classic. 
```

Comment:
This (appropriately) reads quite historical, and yet also says "new".
Given that v155 isn't new, nor is the idea of "responsive design", I wonder if wording this paragraph something like the following (which is more just matter-of-fact than storytelling) would make better sense in the docs?

The "Responsive Classic" template was added as the main template in Zen Cart 1.5.5.
It uses techniques like browser user-agent-detection to adapt its display for mobile devices like phones and tablets.

Reply:
got it - will add on next checkin.